sanity/schemas/caseStudies.ts

```diff
@@ -0,0 +1,42 @@
+export default {
+  name: 'caseStudies',
+  title: 'Case Studies',
+  type: 'document',
+  fields: [
+    {
+      name: 'project',
+      type: 'reference',
+      to: [
+        {
+          type: 'projects',
+          title: 'Project',
+        },
+      ],
+    },
+    {
+      name: 'mission',
+      title: 'Mission',
+      type: 'string',
+    },
+    {
+      name: 'headline',
+      title: 'Headline',
+      type: 'string',
+    },
```

Comment:
If we're referencing the project at the top, do we need this?

What's a case study? A detailed report of a project, right? 
And if we're already referencing the project to case study, do we need to specify headline specifically?